metaflow/tutorials/00-helloworld/hello.py

```diff
@@ -45,6 +56,7 @@ def end(self):
         print("END step:\n We have reached the END step!\n")
         print("HelloFlow is all done.")
         print("___________________________________________")
+        print(self)
```

Comment:
Not necessarily for this PR but we can move kfp specific examples to a different tutorial folder. I previously created `tutorials/09-hellokfp`

Reply:
Fixed! see https://github.com/Netflix/metaflow/compare/master...zillow:tz/foreach for diff from `tz/foreach` to Netflix Metaflow master origin.